modules/landing_zone/terraform.tf

```diff
@@ -0,0 +1,3 @@
+terraform "backend" "local" {
+  path = "/tmp/.terrahub/local_backend/terraform-google-project-landing_zone/landing_zone/terraform.tfstate"
```

Comment:
please remove `google-project` from here ... this is `aws-landing-zone` project ;)